### PR TITLE
Article Summary

### LINKED ISSUE #24: Create Summary Component
Including:

- [ ] article title
- [ ] article URL
- [ ] article summary
- [ ] highlighted keywords

Response object will look like:
```
{
  'url':'http://bbc.co.uk',
  'article' : {
    'header' : 'An Article Title',
    'summary' : 'The Article Summary',
    'keywords' : ['Boris Johnson', 'Brexit']
  },
  'results' : [
    { 'type' : 'credibility' ...... },
    { 'type' : 'polarity' ..... },
    { 'type' : 'subjectivity' .....},
    { 'type' : 'biasscore' .....}
  ]
}
```

### PR DESCRIPTION
**Pull request by abirdonwire**
Created a layout for the article title, summary and key words for when we have Textblob data.

### CODE REVIEW

**Review by ANA-POTJE — APPROVED** [review]

File: src/components/Results/ArticleSummary.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Grid, Typography, makeStyles } from "@material-ui/core/";
+import "../../App.css";
```
Comment (jsjohnstone) [inline src/components/Results/ArticleSummary.js:3]: @abirdonwire Does app.css need to be re-imported? I thought it'd be 'applied' by app.js and also apply to any child components?

File: src/components/Results/ArticleSummary.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        </Grid>
+        <Grid item xs={12}>
+          <Typography variant="body2" className={classes.SummaryText}>
+            The summary of the article.The summary of the article.The summary of
+            the article.The summary of the article.The summary of the article.
+          </Typography>
+        </Grid>
+        <Grid item xs={12}>
+          <Typography variant="body1">
+            Keywords: Boris Johnson, Brexit, Economy
```
Comment (jsjohnstone) [inline src/components/Results/ArticleSummary.js]: What if we used Material 'Chips' to show these keywords?
https://material-ui.com/components/chips/

**Review by jsjohnstone — COMMENTED** [review]
I think this looks awesome! I've got one question (about app.css) and one suggestion (About using chips to display the individual keywords), but think it looks great! :) 

Let me know your thoughts on those and I'll approve 👍

File: src/components/Results/ArticleSummary.js (hunk 1 above)
Comment (abirdonwire) [reply src/components/Results/ArticleSummary.js:3]: Hmm, I don't know! I'll investigate

File: src/components/Results/ArticleSummary.js (hunk 2 above)
Comment (abirdonwire) [reply src/components/Results/ArticleSummary.js]: Yes! Loving these.

Comment (abirdonwire) [reply src/components/Results/ArticleSummary.js]: I like these! The Basic chips ought to do the trick.

File: src/components/Results/ArticleSummary.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+function ArticleSummary() {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <Grid
+        container
+        justify="flex-start"
+        direction="column"
+        className={classes.ArticleSummaryText}
```
Comment (Jyothsna-GC) [inline src/components/Results/ArticleSummary.js:41]: ArticleText in place of summary as the analysis is on article

File: src/components/Results/ArticleSummary.js (hunk 2 above)
Comment (Jyothsna-GC) [inline src/components/Results/ArticleSummary.js]: Highlights could replace - keyword

**Review by Jyothsna-GC — APPROVED** [review]

File: src/components/Results/ArticleSummary.js (hunk 2 above)
Comment (jsjohnstone) [reply src/components/Results/ArticleSummary.js]: Yeah! Or if they take up too much room, the `<Chip size="small" />` is quite tidy as well.

File: src/components/Results/ArticleSummary.js (hunk 3 above)
Comment (abirdonwire) [reply src/components/Results/ArticleSummary.js:41]: This is the className for the container box of the all  the text. Is that what you mean?

File: src/components/Results/ArticleSummary.js (hunk 2 above)
Comment (abirdonwire) [reply src/components/Results/ArticleSummary.js]: I think I prefer Keywords, as these words are the search terms themselves.

Comment (abirdonwire) [reply src/components/Results/ArticleSummary.js]: Chips ahoy!

**Pull request merged**